docs/components/base/fake.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Configure a Fake Base"
+linkTitle: "fake"
+weight: 50
+type: "docs"
+description: "Configure a fake base to use for testing without physical hardware."
+tags: ["base", "components"]
+# SMEs: Steve B
+---
+
+Configure a `fake` base to test implementing a base component on your robot without any physical hardware.
```

Comment:
```suggestion
You can use the `fake` base model to test implementing a base component on your robot without any physical hardware.
```
Just so you don't repeat "Configure a `fake` base" in two consecutive sentences

Reply:
This matches the arm page! Helpful feedback but I am fine with not making the change and keeping the pages somewhat robotic but straightforward for the moment

Reply:
Though we are a robotics company :p, I'd advocate for changing the wording both here and on the arm page. It just looks like a copy paste error to me if that makes sense.